_posts/2018-09-05-Data-Continuity-Service-for-DynamoDB.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: post
+title: Data Continuity Service for DynamoDB
+banner: /assets/posts/2018-09-05-Data-Continuity-Service-for-DynamoDB/solution-overview.png)
+author: tvb
+---
+
+This is the first post of three where I am going to explain how to build and configure a Data Continuity Service (DCS) for Amazon DynamoDB by using Amazon DynamoDB Streams, AWS Lambda and the Amazon S3 services.
```

Comment:
> where I am going

I think you said others will write posts as well, so that would be a `we are`.

Reply:
Thanks